src/DynamoUtilities/PIIDetector.cs

```diff
@@ -119,15 +119,13 @@ internal static string RemovePIIData(string data)
         {
             string result;
 
-            // ToDo: some of the regex patterns are interfering with other data in the json,
-            // so commenting them for now and will address those patterns later.
             result = Regex.Replace(data, emailPattern, "");
             result = Regex.Replace(result, directoryPattern, "");
             result = Regex.Replace(result, ssnPattern, "");
             result = Regex.Replace(result, datePattern, "");
-            //result = Regex.Replace(result, websitePattern, "");
-            //result = Regex.Replace(result, creditCardPattern, "");
-            //result = Regex.Replace(result, ipPattern, "");
+            result = Regex.Replace(result, ipPattern, "");
+            result = Regex.Replace(result, creditCardPattern, "");
+            result = Regex.Replace(result, websitePattern, "");
```

Comment:
hi @reddyashish So the trick is to just change the order of regex?

Reply:
The main change is to use RemovePIIData(JObject jsonObject) API on the whole json object which will look for the regex pattern in the Views, Annotations etc.

Reply:
Right, in the past we are running the regex against the text?

Reply:
yes